test/QuickTest/project.json

```diff
@@ -4,12 +4,15 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1-*"
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*"
   },
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
       "dependencies": {
+        "Microsoft.NETCore.App": {
+          "version": "1.0.0-*",
+          "type": "platform"
+        },
         "System.Console": "4.0.0-*"
```

Comment:
No, this comes from `Microsoft.NETCore.App`, my automation doesn't know how to remove dependencies.